SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -404,41 +412,51 @@ bool FLocalDeploymentManager::TryStopLocalDeploymentGracefully()
 		return TryStopLocalDeployment();
```

Comment:
We should probably lock the critical section in TryStopLocalDeployment as well. It too has the potential to be running while the FLocalDeploymentManager is being destroyed.

Reply:
good point, will add